src/cmdline.js

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
```

Comment:
If at all possible, it'd be much better if a node module could use the same code as the existing command line tool. (DRY)